iml-system-test-utils/src/lib.rs

```diff
@@ -373,10 +373,19 @@ pub async fn wait_on_services_ready(config: &Config) -> Result<(), TestError> {
 pub async fn setup_bare(config: Config) -> Result<Config, TestError> {
     vagrant::up()
         .await?
-        .arg(config.manager)
+        .arg(config.iscsi)
         .checked_status()
         .await?;
 
+    vagrant::up_parallel(
```

Comment:
Does vagrant support bringing up multiple nodes in parallel?

Reply:
Nope